homeassistant/components/zwave_js/strings.json

```diff
@@ -144,5 +144,11 @@
       "ping": "Ping device",
       "reset_meter": "Reset meters on {subtype}"
     }
+  },
+  "issues": {
+    "invalid_server_version": {
+      "title": "Newer version of Z-Wave JS Server needed",
+      "description": "The version of Z-Wave JS Server you are currently running is too old for this version of Home Assistant, please update the Z-Wave JS Server to the latest version."
```

Comment:
Is there a way to get the required Z-Wave JS version that we need, then insert that here? I might be overthinking things, but that would be a very explicit instruction to the user.

That said, if the latest is good enough, ignore more. 😆

Reply:
Unfortunately not. I can provide a little more info in the next lib release but the problem is that the versions required will be installation method dependent.